CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+All official releases can be found on this repository's [releases page](https://github.com/ChartBoost/chartboost-mediation-android-adapter-digital-turbine-exchange/releases).
+
+### 4.8.3.1.0
+- This version of the adapter has been certified with Digital Turbine Exchange SDK 8.3.1.
+
```

Comment:
Should this be below the 5.x entry?

Reply:
it's for DTE 8.3.1 though. presumably we'll release 5.8.3.1.0 later